### ISSUE TITLE
Test pipeline

### ISSUE THREAD

**Issue by MarcusGoldschmidt** [Kindelia/Kindelia#85]
Would be nice if we have a pipeline for testing and build the application for multiple targets.

**Comment by steinerkelvin** [comment]
Indeed, that's a strong ideia. We also have some code at https://github.com/Kindelia/HVM/blob/master/tests/test_cli.py to run black box tests that we could use to test `kindelia run` on the files at `/test/` and more.
